mantidproject/mantid
# Control the intregration slider using arrow keys

**Description of work.**
This PR allows to control the integration slider, in the instrument view, using the left and right arrow keys.


**To test:**
- On workbench, load a non-monochromatic file, such as `ExternalData/Testing/Data/UnitTest/ILL/D17/397806.nxs`.
- Open the instrument window, and play with the integration bar. In particular, check the view is only updated when the key is released.
 


Fixes #28342. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
